test/extended/util/docker.go

```diff
@@ -0,0 +1,85 @@
+package util
+
+import (
+	"fmt"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	tutil "github.com/openshift/origin/test/util"
+)
+
+func TagImage(tagee, tagor string) error {
```

Comment:
godoc (also for the rest of the functions). This stuff will be re-used in many test cases, so deserves documentation.